internal/net/grpc/stream.go

```diff
@@ -164,6 +143,7 @@ func BidirectionalStreamClient(stream grpc.ClientStream,
 	}))
 
 	defer func() {
+		status.FromError(err)
```

Comment:
Is this needed?

Reply:
Sorry it's miss paste. I'll remove it later